subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/report/IndexPageGenerator.java

```diff
@@ -152,4 +155,12 @@ protected void renderScenarioButtons(int index, ScenarioBuildResultData scenario
             }
         };
     }
+
+    private boolean isSmallRegression(ScenarioBuildResultData scenario) {
+        return scenario.getRawData().stream()
+            .filter(ScenarioBuildResultData::isRegressed)
+            .map(it -> flakinessDataProvider.getScenarioRegressionResult(it.getScenarioName(), it.getDifferencePercentage()))
+            .anyMatch(it -> it == PerformanceFlakinessDataProvider.ScenarioRegressionResult.SMALL_FLAKY_REGRESSION);
```

Comment:
```suggestion
            .anyMatch(SMALL_FLAKY_REGRESSION::equals);
```

And static import it?

Reply:
I am using `failsBuild` now.